docs/environment.yaml

```diff
@@ -9,4 +9,4 @@ dependencies:
 - sphinx
 - openmm
 - pip:
-  - git+https://github.com/OpenFreeEnergy/ofe-sphinx-theme@main
+  - git+https://github.com/OpenFreeEnergy/ofe-sphinx-theme@a45f3edd5bc3e973c1a01b577c71efa1b62a65d6
```

Comment:
This is a known issue upstream

Reply:
Can you link to the issue? I'd like to learn what's going on here so we can follow up later.

Reply:
Here we go: https://github.com/OpenFreeEnergy/ofe-sphinx-theme/issues/5

It's a fixaeble packaging issue but I've not had time to look into it.